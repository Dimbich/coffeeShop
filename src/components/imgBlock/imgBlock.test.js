import React from 'react';
import ImgBlock from './imgBlock';
import renderer from 'react-test-renderer';



describe('Testing <ImgBlock/>', () => {
  it('ImgBlock have rendrered correctly', () => {
    const imgBlock = renderer.create(<ImgBlock/>).toJSON();
    expect(imgBlock).toMatchSnapshot();
  })
})  