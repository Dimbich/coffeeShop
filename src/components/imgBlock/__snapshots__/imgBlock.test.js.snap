// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <ImgBlock/> ImgBlock have rendrered correctly 1`] = `
<img
  className="sc-bdVaJa bwHsYK"
  src="undefined"
/>
`;
